README.md

```diff
@@ -122,25 +152,37 @@ Given the following DOM tree
 </div>
 ```
 
-we could find all of the elements with `className === 'banner'` by calling `document.getElementsByClassName('banner')`.
+we could find all of the elements with `className === 'banner'` by calling
+`document.getElementsByClassName('banner')`.
 
 **Try it out!**
 
-Inspect the web page again, this time making note of a className. Get all elements with that className and give 'em a look. Remember, you can use a `for` loop to loop through them. (You can also assign the return value of `document.getElementsByClassName()` to a variable: `var elements = document.getElementsByClassName('banner')`.)
+Inspect the web page again, this time making note of a className. Get all
+elements with that className and give 'em a look. Remember, you can use a `for`
```

Comment:
I'd cut looping here...but i do like using var assignment...maybe as a promissory note for later things? See earlier comment. We might prefer modularity.